site/docs/master/aws-config.md

```diff
@@ -148,11 +148,12 @@ Install Velero, including all prerequisites, into the cluster and start the depl
 
 ```bash
 velero install \
-    --provider aws \
+    --provider velero.io/aws \
```

Comment:
Does this change imply that users are going to be required to change their existing BSL provider names?

Reply:
Yes, they'll have to change if we want to make our plugins play by the same rules as everyone else.

IMO we should provide a translation script or command for upgrading. It shouldn't be all that involved to fetch each BSL and update the `provider` field.